feeds/wifi-ax/hostapd/patches/f00-019-channel-switch-event.patch

```diff
@@ -0,0 +1,64 @@
+--- a/src/ap/ubus.c
++++ b/src/ap/ubus.c
+@@ -531,6 +531,7 @@ enum {
+ 	CSA_VHT,
+ 	CSA_HE,
+ 	CSA_BLOCK_TX,
++	CSA_REASON,
+ 	__CSA_MAX
+ };
+ 
+@@ -545,6 +546,7 @@ static const struct blobmsg_policy csa_p
+ 	[CSA_VHT] = { "vht", BLOBMSG_TYPE_BOOL },
+ 	[CSA_HE] = { "he", BLOBMSG_TYPE_BOOL },
+ 	[CSA_BLOCK_TX] = { "block_tx", BLOBMSG_TYPE_BOOL },
++	[CSA_REASON] = { "reason", BLOBMSG_TYPE_INT32 },
+ };
+ 
+ #ifdef NEED_AP_MLME
+@@ -810,7 +812,7 @@ void hostapd_dfs_csa_timeout(void *eloop
+ 		   iface->conf->ieee80211ac, iface->conf->ieee80211ax);
+ 
+ 	ret = hostapd_enable_iface(iface);
+-	if (ret == 0)
++	if (ret == 0 && css->reason == 0)
```

Comment:
Do we need css->reason == 0 check here?

Reply:
I believe so. This function gets called from `hostapd_switch_chan` when we're trying to switch to a dfs channel. So we need to make sure it gets suppressed for that case too and not just the regular channel hops.